apps/crn-server/src/data-providers/teams.data-provider.ts

```diff
@@ -128,6 +128,7 @@ export class TeamSquidexDataProvider implements TeamDataProvider {
     const inputTeam: InputTeam['data'] = {
       applicationNumber: { iv: input.applicationNumber },
       displayName: { iv: input.displayName },
+      inactiveSince: { iv: input.inactiveSince || null },
```

Comment:
setting to null when it does not have a value because this is the way it comes from squidex